packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -5,11 +5,16 @@ import {
   CreateDBInstanceCommandInput,
   DBInstance,
   DeleteDBInstanceCommand,
+  DescribeDBClustersCommand,
   waitUntilDBInstanceAvailable,
   CreateDBClusterMessage,
   waitUntilDBClusterAvailable,
   DeleteDBClusterCommand,
   DeleteDBClusterCommandInput,
+  $Command,
+  DescribeDBInstancesCommand,
+  DescribeDBClustersCommandOutput,
+  DescribeDBInstancesCommandOutput,
```

Comment:
nit here and throughout: Please sort the imports :) It makes it easier to keep track of what is already imported